### PR TITLE
i18n(zh-cn): update `reference/errors/collection-does-not-exist-error.mdx`

### PR DESCRIPTION
**Pull request by 100gle**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

update `reference/errors/collection-does-not-exist-error.mdx`

#### Related Issue / Implementation PR

#4797

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/collection-does-not-exist-error.mdx [hunk 1]
```diff
@@ -4,6 +4,10 @@ i18nReady: true
 githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
 ---
 
+:::caution[已废弃]
+不存在的集合将不再导致错误，而是忽略警告。
```
Comment (Genteure) [inline src/content/docs/zh-cn/reference/errors/collection-does-not-exist-error.mdx:8]: 原文好像写错了，应该是“输出警告”。

Comment (Genteure) [reply src/content/docs/zh-cn/reference/errors/collection-does-not-exist-error.mdx:8]:
```suggestion
不存在的集合将不再发生错误，而是给出警告。
```

**Comment by Genteure** [comment]
Waiting for https://github.com/withastro/astro/pull/8708 and the bot to sync changes into here.

**Comment by 100gle** [comment]
https://github.com/withastro/astro/pull/8708 has been merged. @Yan-Thomas this PR can go ahead.

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**